### ISSUE TITLE
Additional Functionality

### ISSUE THREAD

**Issue by kunal-vaghela-digiwagon** [vritravaibhav/usbwebcamera#2]
Is there possible to add functionality for zoom in, zoom out, explosure, white balance and contras and other features like that

**Comment by vaibhaveevv** [comment]
You may use flutter ImageFilter widget.

**Comment by kunal-vaghela-digiwagon** [comment]
> You may use flutter ImageFilter widget.

I want to show they filters on camera screen, when i wrap to ImageFilter then i click picture they captured normal image because in camera getting normal view so they click as it is

**Comment by vaibhaveevv** [comment]
Yeah. Actually as much I remember there's no such functionality. you can use also screenshot package it takes screenshot of widget
